docs/blog/introduction-0.15.0/index.qmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "Great Tables `v0.15.0`: Flags, Icons, and Other Formatting Goodies"
+html-table-processing: none
+author: Rich Iannone
+date: 2024-12-19
+freeze: true
+jupyter: python3
+---
+
+The development of Great Tables is really moving along these days. We just released version `0.15.0` and it adds quite a few nice things to the package. The features we'll highlight in this post are:
+
+- adding flag icons with the new `fmt_flag()` method
+- peppering your table cells with Font Awesome icons via `fmt_icon()`
+- support for displaying accounting notation with four number-based formatting methods
+
+Let's look at each of these in turn!
+
+### Using `fmt_flag()` to incorporate country flag icons
+
+When tables contain country-level data, having a more visual representation for a country can help the reader more quickly parse the table contents. The new `fmt_flag()` method makes this easy to accomplish. You just need to have either [two-letter country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) or [three-letter country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) in a column.
+
+Here's an example where country flags, shown as simplified circular icons, can be added to a table with `fmt_flag()`:
+
+```{python}
+from great_tables import GT
+from great_tables.data import peeps
+import polars as pl
+
+peeps_mini = (
+    pl.from_pandas(peeps)
+    .filter(pl.col("dob").str.slice(offset=0, length=4) == "1988")
+    .with_columns(name=pl.concat_str(pl.col("name_given") + " " + pl.col("name_family")))
```

Comment:
I think you can remove the `pl.concat_str()` call here

Reply:
Done, thanks!